spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -153,6 +153,21 @@ def schedule_for_future
       payload = hash_including(update_type: 'major', change_note: 'This is a public change note.')
       expect(Services.publishing_api).to have_received(:put_content).with(step_by_step_page.content_id, payload)
     end
+
+    it "creates an internal change note for minor change" do
+      schedule_for_future
+
+      expected_description = "Scheduled by Name Surname for publishing at 10:26am on 20 April 2030"
+      expect(step_by_step_page.internal_change_notes.first.description).to eq expected_description
+    end
+
+    it "creates an internal change note for major change" do
```

Comment:
I'd take inspiration from the scenario names from the `publish` tests here, i.e.

`it "generates an internal change note with change note text for major changes"`

and

`it "generates an internal change note without change note text for minor changes"`

Reply:
I'll update the major change description.  I'll leave the minor change description since the description is technically correct (i.e. a change note has been created).